### DISCUSSION TITLE
Streamlink still gives me ads even with a VPN from Ukraine?

### DISCUSSION THREAD

**Discussion by Chris763454357** [streamlink/streamlink-twitch-gui#994] [Q&A] [answered]
I recently found out some countries have little to no ads with twitch. Two days ago I got a VPN. 

VPN from Ukraine:
- So far not a single pre-rolls or mid rolls when viewing twitch in my browser.
- Streamlink Twitch GUI: Continues to show me the "commercial break in progress" purple screen during ads and the 'getting stream ready' message when starting a stream without mid-rolls.

Any insight on why this is happening?

**Comment by bastimeyer** [comment, marked as answer]
Streamlink Twitch GUI is a frontend for Streamlink. It has nothing to do with anything streaming related.

If you're seeing the purple screen, then you have not set the [`--twitch-disable-ads` parameter](https://streamlink.github.io/cli.html#twitch) or the according checkbox in the Twitch GUI settings menu.

Read this:
https://streamlink.github.io/cli/plugins/twitch.html

And if you're seeing ads or the purple screen while using a VPN, then this means that Twitch must've made some changes. I can't fix this.
